core/src/validator.rs

```diff
@@ -449,11 +451,16 @@ impl Validator {
 
         *start_progress.write().unwrap() = ValidatorStartProgress::StartingServices;
 
-        verify_udp_stats_access().unwrap_or_else(|err| {
-            error!("Failed to access UDP stats: {}", err);
-            abort();
-        });
-        let system_monitor_service = Some(SystemMonitorService::new(Arc::clone(&exit)));
+        if !config.no_os_network_stats_reporting {
+            verify_udp_stats_access().unwrap_or_else(|err| {
+                error!("Failed to access UDP stats: {}. Bypass check with --no-os-network-stats-reporting.", err);
+                abort();
+            });
+        }
+        let system_monitor_service = Some(SystemMonitorService::new(
```

Comment:
Why start the service at all when `config.no_os_network_stats_reporting` is false?

Reply:
We could also turn off memory reporting, copy the code, or move memory reporting elsewhere.

Reply:
I think it's ok as is, was just asking